### DISCUSSION TITLE
Gateway mirror problem

### DISCUSSION THREAD

**Discussion by cham-d** [linkerd/linkerd2#7415] [Q&A]
Hey, i am new in kubernetes and linkerd so take it easy :)
I am trying to implement a multi cluster implementation locally (kinD), using the official [tutorial](https://linkerd.io/2.10/tasks/multicluster/)
I installed Metallb in both of the clusters,so the linkerd-gateway can take an ip.
I linked west and east cluster, using the guide's command,but when i run the check command
( `linkerd --context=kind-west multicluster check ` ), everything is ok, except this:
`× all gateway mirrors are healthy`
 `probe-gateway-kind-east.linkerd-multicluster mirrored from cluster [kind-east] has no endpoints`
What am i doing wrong?
A part of Logs from service-mirror:
`k logs linkerd-service-mirror-east-9c76864cb-mkqvn service-mirror -n linkerd-multicluster --context kind-west   `
`time="2021-12-07T18:20:03Z" level=info msg="Link watch terminated; restarting watch"
time="2021-12-07T18:20:03Z" level=info msg="Got updated link east: {Name:east Namespace:linkerd-multicluster TargetClusterName:east TargetClusterDomain:cluster.local TargetClusterLinkerdNamespace:linkerd ClusterCredentialsSecret:cluster-credentials-east GatewayAddress:172.18.255.200 GatewayPort:4143 GatewayIdentity:linkerd-gateway.linkerd-multicluster.serviceaccount.identity.linkerd.cluster.local ProbeSpec:ProbeSpec: {path: /ready, port: 4191, period: 3s} Selector:{MatchLabels:map[] MatchExpressions:[{Key:mirror.linkerd.io/exported Operator:Exists Values:[]}]}}"
time="2021-12-07T18:20:03Z" level=error msg="Unable to create cluster watcher: cannot connect to api for target cluster remote: Get \"https://127.0.0.1:45333/version?timeout=32s\": dial tcp 127.0.0.1:45333: connect: connection refused"
time="2021-12-07T18:20:13Z" level=info msg="Link watch terminated; restarting watch"
time="2021-12-07T18:20:13Z" level=info msg="Got updated link east: {Name:east Namespace:linkerd-multicluster TargetClusterName:east TargetClusterDomain:cluster.local TargetClusterLinkerdNamespace:linkerd ClusterCredentialsSecret:cluster-credentials-east GatewayAddress:172.18.255.200 GatewayPort:4143 GatewayIdentity:linkerd-gateway.linkerd-multicluster.serviceaccount.identity.linkerd.cluster.local ProbeSpec:ProbeSpec: {path: /ready, port: 4191, period: 3s} Selector:{MatchLabels:map[] MatchExpressions:[{Key:mirror.linkerd.io/exported Operator:Exists Values:[]}]}}"
time="2021-12-07T18:20:13Z" level=error msg="Unable to create cluster watcher: cannot connect to api for target cluster remote: Get \"https://127.0.0.1:45333/version?timeout=32s\": dial tcp 127.0.0.1:45333: connect: connection refused"
`
Thank you.

**Comment by wmorgan** [comment]
I think this is an issue with the way that Kind sets up access to the API server. You need to use the `--api-server-address` to tell Linkerd how to access the API server. There is an example here: https://github.com/olix0r/l2-k3d-multi/blob/master/link.sh

Sorry, we really need to improve the docs about this.
